frontend/src/components/common/podcast-filter/podcast-filter.tsx

```diff
@@ -0,0 +1,78 @@
+import { Path, FieldValues } from 'react-hook-form';
```

Comment:
this component should be as popup
logic for opening/closing should be in the homapage component